README.md

```diff
@@ -37,6 +38,49 @@ This package is a work in progress:
 - JavaScript/TypeScript package, CLI
 - TODO: API and web UI
 
+### Provider and Model Matching
+
+The library automatically handles provider aliases and model name variations through YAML-based configuration. This allows you to use various provider identifiers and model names while the library maps them to the correct providers and models.
+
+**Provider Aliases:**
+
+Provider aliases are defined in the YAML configuration files and support the same matching logic as models:
+
+- **Google**: `google`, `gemini`, `google-gla`, `google-vertex`, `google-ai`
+- **Meta**: `meta`, `meta-llama`, `llama`
+- **Mistral**: `mistral`, `mistralai`
+- **Anthropic**: `anthropic`, `claude`
+- **OpenAI**: `openai`, `gpt`
+
+**Model Normalization:**
```

Comment:
I thought we removed this?

Reply:
yes, this is rudiment, didn't actualise readme yet

Reply:
removed from readme as well